spec/models/cell_spec.rb

```diff
@@ -10,6 +10,10 @@ module Alchemy
       it "should return an Array" do
         expect(Cell.definitions).to be_a(Array)
       end
+
+      it "should allow erb generated definitions" do
+        expect(Cell.definitions.collect {|d| d['name']}).to include('erb_cell')
```

Comment:
Prefer `map` over `collect`.<br>Space between { and | missing.<br>Space missing inside }.